### PR TITLE
Ft/skills

### PR DESCRIPTION
**Pull request by AdelineA**
- add header
- add a team photo
- add team description
- add mobile responsiveness
- add medium screen responsiveness
- add desktop responsiveness

### CODE REVIEW

File: build/sections/skills.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<div class="container">
+    <div class="font-bold text-xl lg:text-[26px] pt-16 text-white">
+        <h1>Eine mögliche T<span class="border-b-4 border-primary-green">eamkonstellation</h1></span>
```
Comment (gsngdavid) [inline build/sections/skills.html]: Few changes.🤏

- Use right font-family (Raleway)
- Use right underline style ( check figma design)

**Review by gsngdavid — CHANGES_REQUESTED** [review]
Almost there, just a few changes🤏


https://github.com/UmutesiMMA/ojembaProject/assets/89668775/d7fdb6c8-74b8-48bb-9bcd-951d959f880c


1. Title gets out of the container when screen is zoomed out

Screenshot 2023-08-21 174051

2. Blueish background

Screenshot 2023-08-21 173343

**Figma**:
Screenshot 2023-08-21 173408

3. Optional: Consinder centering images in small screen
Screenshot 2023-08-21 174458

**Review by UmutesiMMA — CHANGES_REQUESTED** [review]
image

there's an extra black line that I highlighted in red. I suggest you remove it as it was't part of the figma design , I put a screenshot of the figma design for reference

Optional: may you increase the gap between the columns? I highlighted in green the slight difference between what you have and what was in the figma design
image

**Review by UmutesiMMA — APPROVED** [review]

**Review by gsngdavid — APPROVED** [review]
well-done-despicable-me

**Pull request merged**